frappe/boot.py

```diff
@@ -442,6 +443,28 @@ def load_currency_docs(bootinfo):
 	bootinfo.docs += currency_docs
 
 
+def get_marketplace_apps():
+	import requests
+
+	apps = []
+	cache_key = "marketplace_apps"
+
+	def get_apps_from_fc():
+		remote_site = frappe.conf.frappecloud_url or "frappecloud.com"
+		request_url = f"https://{remote_site}/api/method/marketplace-apps"
+		request = requests.get(request_url, timeout=5.0)
```

Comment:
If this endpoint goes down, we could end up DDOSing ourselves :woozy_face: Can't think of any good alternative right now though, we'll rate limit it ig?